mlrun/config.py

```diff
@@ -131,6 +130,27 @@ def dump_yaml(self, stream=None):
     def reload():
         _populate()
 
+    @property
+    def version(self):
+        # importing here to avoid circular dependency
```

Comment:
empty line before comment

Reply:
IIRC we decided that this one won't be a convention in MLRun, because it has some compatibility issues with Black (formatter)